numpy/array_api/_typing.py

```diff
@@ -50,7 +50,7 @@ class NestedSequence(Protocol[_T_co]):
     def __getitem__(self, key: int, /) -> _T_co | NestedSequence[_T_co]: ...
     def __len__(self, /) -> int: ...
 
-Device = Literal["cpu"]
+Device = type(CPU_DEVICE)
```

Comment:
Dynamically assigning a type like this isn't gona work I'm afraid. Probably want to go with an `Any` here.
```suggestion
Device = Any
```

Reply:
We do know the type https://github.com/numpy/numpy/pull/25370/files#diff-79e6f716d807a9f60474911f90c087a717802dd6e63c82cd11689b7e47e943d1R45. I just figured this would work.